changelog.d/gh-9987.changed

```diff
@@ -0,0 +1,3 @@
+Changed several secrets related feature flags.
+* The `--beta-testing-secrets-enabled` option, deprecated for several months, is now removed. Use `--secrets` as its replacement.
+* The `--secrets` option removed from scan command. `--config secrets` is the way to run secrets with `semgrep scan`.
```

Comment:
May be better to just have a changelog entry for each, given how these get compiled.

Reply:
Done in [d01fabf](https://github.com/semgrep/semgrep/pull/9987/commits/d01fabfa2934bda6074c36c98546de202232009c)